src/IECoreHoudini/GEO_CobIOTranslator.cpp

```diff
@@ -163,3 +164,27 @@ GA_Detail::IOStatus GEO_CobIOTranslator::fileSaveToFile( const GEO_Detail *geo,
 	
 	return fileSaveToFile( geo, fileName );
 }
+
+bool GEO_CobIOTranslator::fileStat( const char *fileName, GA_Stat &stat, uint level )
+{
+	try
+	{
+		ReaderPtr reader = Reader::create( fileName );
+		if ( !reader )
+		{
+			return false;
+		}
+		
+		CompoundObjectPtr header = reader->readHeader();
+
+		UT_BoundingBox bbox = convert<UT_BoundingBox>( reader->readHeader()->member<const Box3fData>( "bound", true )->readable() );
```

Comment:
Oh sorry, I made a wrong assumption without checking the header. I thought the 2nd arg to `member` as a `createIfMissing` arg. Your way should work.